resources/lang/en/admin/hardware/general.php

```diff
@@ -45,4 +45,5 @@
     'alert_details' => 'Please see below for details.',
     'custom_export' => 'Custom Export',
     'mfg_warranty_lookup' => ':manufacturer Warranty Status Lookup',
+    'user_department' => 'User Department: ',
```

Comment:
We actually try not to include the colon in the translations these days and instead put it in the blade IIRC

Reply:
fixed. saw a few others that had this formatting. I didn't change them, but will make sure NEW translations put the colon on the blade

Reply:
Yeah, there are a few places where it's inconsistent :(